### PR TITLE
adding hcp/alde-cloud-test

### PR DESCRIPTION
**Pull request by LucasPelegrin**
Adding the alde configuration TOSCA implementation

### CODE REVIEW

File: hpc/alde-cloud-test/input.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+alde_host: http://77.231.202.137:5000
```
Comment (dradX) [inline hpc/alde-cloud-test/input.yaml:1]: Usually it's not a good idea to put real IP addresses in the inputs

File: hpc/alde-cloud-test/service.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      type: string
+    testbed_id:
+      type: string
+
+
+  node_templates:
+    local-workstation:
+      type: tosca.nodes.Compute
+      attributes:
+        private_address: localhost
```
Comment (dradX) [inline hpc/alde-cloud-test/service.yaml:41]: To run this you need to locally install an alde instance with certain setup - as I understand.

**Review by dradX — CHANGES_REQUESTED** [review]
To run this example we probably need to install an alde instance locally and configure it to connect to a HPC login node. Maybe adding a readme.md document would be appropriate to show the prerequisites and setup to run this example.

**Comment by LucasPelegrin** [comment]
The Tosca is executing locally the implementation of the software component... That means that TOSCA is calling the ansible automatism and not creating resources indeed. The ansible script is just configuring the already existing ALDE server which is already configured to execute tasks in the HCP environment. That is the reason why it contains the public Instances of the existing ALDE enpoint.

I am wondering if to create a local ALDE server, how would It be required to configure it to the HCP cluster if an SSH key is required to finish the setup. Therefore the only way to create a fully functional ALDE that will be able to lunch Torque HCP workload it would be to create a custom HCP environment as well...

Therefore the current TOSCA Alde implementation is made just to deploy an app in the existing ALDE server.

It seems a good idea to remove the public IP of the ALDE, better to not expose that information.

**Pull request closed without merging**